Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -375,21 +375,28 @@ struct RGroupDecompData {
 
   // compute the number of rgroups that would be added if we
   //  accepted this permutation
-  int compute_num_added_rgroups(std::vector<size_t> &tied_permutation) {
-    int num_added_rgroups = 0;
+  size_t compute_heavy_rgroup_counts(
```

Comment:
Not quite sure why the function name was changed.

Reply:
@bp-kelley That's because I had originally made it such that it returned the heavy R-group count vector. Then I realized that I also still needed the number of added R-groups, so I passed the vector as reference, but I didn't revert the function name. I have done it now.